--- configure.in~	Sun Jul 21 23:03:06 2002
+++ configure.in	Sun Jul 21 23:03:17 2002
@@ -149,15 +149,15 @@
 		OPENGL_LIBS="$OPENGL_LIBS -L$withval/lib"
 	fi
 	save_CPPFLAGS="$CPPFLAGS"
-	CPPFLAGS="$CPPFLAGS $OPENGL_CFLAGS"
+	CPPFLAGS="$CPPFLAGS $OPENGL_CFLAGS $X_CFLAGS"
 	AC_CHECK_HEADER(GL/gl.h, HAVE_OPENGL=yes, HAVE_OPENGL=no)
 	CPPFLAGS="$save_CPPFLAGS"
 	if test "x$HAVE_OPENGL" = xyes; then
 		AC_CHECK_LIB(GL, glBegin,
-			OPENGL_LIBS="$OPENGL_LIBS -lGL"
+			OPENGL_LIBS="$OPENGL_LIBS -lGL "
 			HAVE_OPENGL=yes,
 			HAVE_OPENGL=no,
-			[$OPENGL_LIBS]
+			[$OPENGL_LIBS $X_LIBS]
 		)
 	fi
 	if test "x$HAVE_OPENGL" != xyes; then
--- src/Makefile.am~	Sat Jul 20 06:30:46 2002
+++ src/Makefile.am	Sun Jul 21 23:14:40 2002
@@ -2,7 +2,7 @@
 
 SUBDIRS = . baseq2 ctf xatrix rogue
 
-std_cflags = -Wall -Werror -pipe @OPT_CFLAGS@
+std_cflags = -Wall -Werror -pipe @OPT_CFLAGS@ -I/usr/X11R6/include
 module_ldflags = -module -avoid-version -rpath $(pkglibdir)
 
 bin_PROGRAMS = quake2
